src/cipher.rs

```diff
@@ -1161,6 +1161,23 @@ impl Castable for rustls_server_cert_verifier {
 }
 
 impl rustls_server_cert_verifier {
+    /// Create a new server certificate verifier that uses the system's root store and WebPKI via
+    /// `rustls-platform-verifier`.
```

Comment:
```suggestion
    /// Create a new server certificate verifier that uses the system platform verifier via
    /// [`rustls-platform-verifier`].
    ///
    /// [`rustls-platform-verifier`]: https://github.com/rustls/rustls-platform-verifier
```

I think webpki is only in play for UNIX systems w/o a native platform verifier (e.g. linux, bsd, but not android). 

Since there's some nuance here I think it might be best to just say it uses the platform verifier crate and point to that crate's README for more information.